packages/sku/lib/certificate.js

```diff
@@ -70,12 +70,12 @@ const createSelfSignedCertificate = () => {
 const generateCertificate = async (certificatePath, certificateDirPath) => {
   const startTime = performance.now();
 
-  const pems = await createSelfSignedCertificate();
+  const pems = createSelfSignedCertificate();
 
-  const certificateDirExists = existsSync(certificateDirPath);
-
-  if (!certificateDirExists) {
-    await mkdir(certificateDirPath);
+  try {
+    await access(certificateDirPath);
+  } catch {
+    await mkdir(certificateDirPath, { force: true, recursive: true });
```

Comment:
Jedi Mind Tricks don't work on me.

```suggestion
    await mkdir(certificateDirPath, { recursive: true });
```